libs/infra/images.py

```diff
@@ -0,0 +1,97 @@
+from dataclasses import dataclass
+
+BASE_IMAGES_DIR = "cnv-tests"
+
+
+@dataclass
+class Cirros:
+    RAW_IMG: str | None = None
+    RAW_IMG_GZ: str | None = None
+    RAW_IMG_XZ: str | None = None
+    QCOW2_IMG: str | None = None
+    QCOW2_IMG_GZ: str | None = None
+    QCOW2_IMG_XZ: str | None = None
+    DISK_DEMO: str | None = None
+    DIR: str = f"{BASE_IMAGES_DIR}/cirros-images"
+    DEFAULT_DV_SIZE: str = "1Gi"
+    DEFAULT_MEMORY_SIZE: str = "64M"
+
+
+@dataclass
+class Rhel:
+    RHEL7_8_IMG: str | None = None
+    RHEL7_9_IMG: str | None = None
+    RHEL8_0_IMG: str | None = None
+    RHEL8_2_IMG: str | None = None
+    RHEL8_8_IMG: str | None = None
+    RHEL8_9_IMG: str | None = None
+    RHEL8_10_IMG: str | None = None
+    RHEL9_3_IMG: str | None = None
+    RHEL9_4_IMG: str | None = None
+    RHEL9_5_IMG: str | None = None
+    RHEL9_6_IMG: str | None = None
+    RHEL8_REGISTRY_GUEST_IMG: str = "registry.redhat.io/rhel8/rhel-guest-image"
+    RHEL9_REGISTRY_GUEST_IMG: str = "registry.redhat.io/rhel9/rhel-guest-image"
+    # TODO: change back to registry.redhat.io when rhel10 is available
+    RHEL10_REGISTRY_GUEST_IMG: str = "registry.stage.redhat.io/rhel10/rhel-guest-image"
```

Comment:
It should be available now

Reply:
should be updated and verified in a separate PR